### ISSUE TITLE
Marp theme folder hierarchy

### ISSUE THREAD

**Issue by nicola-lunghi** [marp-team/marp-vscode#181]
Hi,

I have the following directory structure

```
themes/
    my_theme.css
assets/
    logo.png
    bg.png
presentations/
     doc1/
         assets/
         presentation1.md
         presentation2.md
         presentation3.md
     conference2/
         assets/
         presentation1.md
     conference3/
         assets/
         presentation1.md
```
I want to use in my custom theme some custom images (logo and bg) that are in the main assets folder.

```
background: url(./assets/bg.png)
```

The problem is that it seems that the path for the css are relative to the presentation file and not to the css.
Also there's a way in which I can tell vscode where to put my files instead of selecting every time the folder?
Like to batch make all the files?

Thanks
Nick

**Comment by yhatt** [comment]
The base path of CSS will be always same as `.md` file because CSS theme will bundle to Marp presentation as the inline stylesheet. 

Surely it seems like counterintuitive so preprocessing for relative URL resolution in Marp (or [Marpit framework](https://github.com/marp-team/marpit)) may be good.

These are workarounds:

- Use URI started with `https://` for assets ([Marp recommends to upload assets into online due to prevent security problems by third-party themes](https://github.com/marp-team/marp-cli#security-about-local-files)).
- Add theme CSS preprocessing for encode images into inline base64, [just like Marp Core is doing](https://github.com/marp-team/marp-core/blob/cf4476400cf79794f468d2f3f6ebfe9a974f47a3/rollup.config.js#L44-L48).

If you want advanced conversion and batch processing, we recommend to use [Marp CLI](https://github.com/marp-team/marp-cli). The VS Code extension has intentionally designed for a simple workflow.

**Comment by MiguelDomingues** [comment]
I've came across a similar issue.
To make things simple, I embedded the images (in base64) into a CSS file directly - this is not a perfect solution, it is more a workaround, but works on VSCode and CLI.

Something like this:
```css
background-image: url(data:image/png;base64,----PLAECE HERE THE IMAGE ENCODED IN BASE64----);
```

The CSS will become quite larger with images, specially because base46 encoding uses more bytes
So far, I've not had any performance issues.
